### PR TITLE
Add LIP "Introduce Legacy module"

### PR DESCRIPTION
**Pull request by MaximeGagnebin**

### CODE REVIEW

**Review by IkerAlus — APPROVED** [review]
LGTM.

Great work!

**Review by Incede — APPROVED** [review]

File: proposals/lip-Introduce_Legacy_module.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+The 8-byte legacy address corresponding to the given input.
+
+
+##### Execution
+
+```python
+getLegacyAddress(publicKey):
+    hashedKey = SHA-256(publicKey)
+    firstEightBytes = 8 first bytes of hashedKey
```
Comment (ricott1) [inline proposals/lip-Introduce_Legacy_module.md:133]: I would reverse the code to match the variable name
```suggestion
    firstEightBytes = first 8 bytes of hashedKey
```

File: proposals/lip-Introduce_Legacy_module.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+For a reclaim transaction `trs` to be valid, the legacy accounts substore must contain an entry with store key equal to `getLegacyAddress(trs.senderPublicKey)` and store value `{"balance": trs.params.amount}`, serialized using `legacyAccountsSchema`.
+
+
+##### Execution
+
+When a reclaim transaction `trs` is executed, the following is done:
+
+* Delete the entry from the legacy accounts substore with store key `getLegacyAddress(trs.senderPublicKey)`.
+* Call the function `mint(newAddress, 0, trs.params.amount)`, where `newAddress` is the [20-byte address][LIP18-addressComputation] derived from `trs.senderPublicKey`. 
+  The function `mint` is defined in the [Token module][token-LIP].
```
Comment (ricott1) [inline proposals/lip-Introduce_Legacy_module.md:181]: minor point, but maybe we could link directly to the function definition

**Review by ricott1 — CHANGES_REQUESTED** [review]
minor points

File: proposals/lip-Introduce_Legacy_module.md (hunk 2 above)
Comment (MaximeGagnebin) [reply proposals/lip-Introduce_Legacy_module.md]: done

**Review by ricott1 — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**